buildSrc/src/main/java/dependency-guard.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import com.android.build.gradle.TestedExtension
+import com.dropbox.gradle.plugins.dependencyguard.DependencyGuardPluginExtension
+
+val configurationNames = when {
+  // record the root project's *build* classpath
+  project == rootProject -> listOf("classpath")
+  // Android variants and their configurations are added later in the configuration phase,
+  // so we can't look them up now using the `configurations` property.
+  // Instead we can just hard-code "releaseRuntimeClasspath" for any module which has AGP applied.
+  // This is actually pretty robust, since if this configuration ever changes,
+  // dependency-guard will fail when trying to look it up.
+  extensions.findByType<TestedExtension>() != null -> listOf("releaseRuntimeClasspath")
```

Comment:
This is also eager and could be made lazy, fyi. There's an `extensions.configure(...)` API that acts as a callback instead